src/it/java/org/checkstyle/base/AbstractItModuleTestSupport.java

```diff
@@ -131,6 +131,31 @@ protected final Checker createChecker(Configuration moduleConfig)
         return createChecker(moduleConfig, findModuleCreationOption(name));
     }
 
+    /**
+     * Creates {@link Checker} instance for list  of  {@link Configuration} instances
+     * with parent {@link TreeWalker}.
+     *
+     * @param moduleConfig {@link Configuration} instances.
+     * @return {@link Checker} instance.
+     * @throws Exception if an exception occurs during checker configuration.
+     */
+    protected final Checker createChecker(List<Configuration> moduleConfig)
+            throws Exception {
+        final Configuration dc;
+        dc = createTreeWalkerConfig(moduleConfig);
+        final Checker checker = new Checker();
+
+        // make sure the tests always run with English error messages
+        // so the tests don't fail in supported locales like German
+        final Locale locale = Locale.ENGLISH;
+        checker.setLocaleCountry(locale.getCountry());
+        checker.setLocaleLanguage(locale.getLanguage());
+        checker.setModuleClassLoader(Thread.currentThread().getContextClassLoader());
+        checker.configure(dc);
```

Comment:
Instead I did this one. Basically similar implementation of 
`createChecker(Configuration moduleConfig,ModuleCreationOption moduleCreationOption)`